modin/pandas/io.py

```diff
@@ -103,10 +103,18 @@ def parser_func(
         memory_map=False,
         float_precision=None,
     ):
+        # ISSUE #2408: parse parameter shared with pandas read_csv and read_table and update with provided args
+        _pd_read_csv_signature = dict(inspect.signature(pandas.read_csv).parameters)
         _, _, _, kwargs = inspect.getargvalues(inspect.currentframe())
         if kwargs.get("sep", sep) is False:
             kwargs["sep"] = "\t"
-        return _read(**kwargs)
+
+        shared_keys = set(_pd_read_csv_signature.keys()).intersection(kwargs.keys())
+
+        for key in shared_keys:
```

Comment:
With @amyskov's suggestion, we can use dictionary comprehension and avoid the `intersection` call altogether.

```python
{k: v for k, v in kwargs.items() if k in _pd_read_csv_signature}
```

Reply:
this looks good. changed.